src/hex_verl.erl

```diff
@@ -0,0 +1,205 @@
+%% Vendored from hex_verl v1.0.0, do not edit manually
```

Comment:
Note to self to prop-test this against Elixir's `Version`.

Reply:
Can you include a script for us to vendor this dependency so that it's easily repeatable? You can look at https://github.com/hexpm/hex_core/blob/master/vendor.sh for inspiration.

Reply:
Can do.

Reply:
Oh, hmmm.... did you want a script in hex_core for this? One exists in the verl repo right now.

Reply:
Yeah, can we vendor the tests also?

Reply:
Can do.